packages/repository/test/integration/repositories/relation.factory.integration.ts

```diff
@@ -227,3 +151,131 @@ describe('HasMany relation', () => {
     );
   }
 });
+
+describe('BelongsTo relation', () => {
+  let findCustomerOfOrder: BelongsToAccessor<
+    Customer,
+    typeof Order.prototype.id
+  >;
+
+  before(givenCrudRepositories);
+  before(givenAccessor);
+  afterEach(async function resetRepositories() {
```

Comment:
Dont we want to clean up the repos in the `before()` handler so that we can inspect the records after if tests fail and to start with clean repository instances?

Reply:
Sure! I was copying this code from elsewhere, will fix it.